lib/puppet-languageserver/handlers/pdk_handler.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'pdk'
+
+module PuppetLanguageServer
+module Handlers
+module PdkHandler
+  def self.newClass(name, target_dir)
+
+    unless PDK::CLI::Util::OptionValidator.valid_class_name?(name)
```

Comment:
@glennsarti @rodjek I'm calling the PDK::Generate classes directly here to avoid module dir checks and the CLI infrastructure. This look ok to you?